state-chain/pallets/cf-ingress-egress/src/migrations/add_boost_pools.rs

```diff
@@ -1,153 +0,0 @@
-use crate::*;
-use cf_chains::DepositChannel;
-use frame_support::traits::OnRuntimeUpgrade;
-
-mod old {
-	use super::*;
-
-	#[derive(CloneNoBound, RuntimeDebug, PartialEq, Eq, Encode, Decode, TypeInfo)]
-	#[scale_info(skip_type_params(T, I))]
-	pub struct DepositChannelDetails<T: Config<I>, I: 'static> {
-		pub deposit_channel: DepositChannel<T::TargetChain>,
-		/// The block number at which the deposit channel was opened, expressed as a block number
-		/// on the external Chain.
-		pub opened_at: TargetChainBlockNumber<T, I>,
-		/// The last block on the target chain that the witnessing will witness it in. If funds are
-		/// sent after this block, they will not be witnessed.
-		pub expires_at: TargetChainBlockNumber<T, I>,
-
-		/// The action to be taken when the DepositChannel is deposited to.
-		pub action: ChannelAction<T::AccountId>,
-		/// The boost fee
-		pub boost_fee: BasisPoints,
-	}
-
-	#[frame_support::storage_alias]
-	pub type DepositChannelLookup<T: Config<I>, I: 'static> = StorageMap<
-		Pallet<T, I>,
-		Twox64Concat,
-		TargetChainAccount<T, I>,
-		old::DepositChannelDetails<T, I>,
-		OptionQuery,
-	>;
-}
-
-pub struct Migration<T: Config<I>, I: 'static>(PhantomData<(T, I)>);
-
-impl<T: Config<I>, I: 'static> OnRuntimeUpgrade for Migration<T, I> {
-	fn on_runtime_upgrade() -> Weight {
-		// Add (default) boost status to existing deposit channels:
```

Comment:
We still want this, right? We no longer need to create pools, but we still need to add new fields to all existing `DepositChannelDetails` records.